### ISSUE TITLE
from:now-9999weeks generate error server side

### ISSUE THREAD

**Issue by morucci** [change-metrics/monocle#590]
```
127.0.0.1 - - [18/Aug/2021:14:11:47 +0000] "POST /search/query HTTP/1.1" 200 - "http://monocle.usersys.redhat.com/cpaas?q=from%3Anow-9999weeks" "Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:90.0) Gecko/20100101 Firefox/90.0"                                                                                                                                                                                                              
Error in $.buckets[0].key: parsing Word64 failed, value is either floating or will cause over or underflow -4.4495136e12                                                                                                                                                                                                                                                                                                                    
CallStack (from HasCallStack):                                                                                                                                                                                                                                                                                                                                                                                                              
  error, called at src/Relude/Debug.hs:288:11 in relude-1.0.0.1-91361c589d4deef18a159e6da401598d1acb0ad9a04c9486633606cef4c3f1e2:Relude.Debug                                                                                                                                                                                                                                                                                               
  error, called at src/Monocle/Prelude.hs:143:21 in monocle-0.1.0.0-inplace:Monocle.Prelude                                                                                                                                                                                                                                                                                                                                                 
  getExn, called at src/Monocle/Backend/Queries.hs:124:35 in monocle-0.1.0.0-inplace:Monocle.Backend.Queries                                                                                                                                                                                                                                                                                                                                
Error in $.buckets[0].key: parsing Word64 failed, value is either floating or will cause over or underflow -4.4495136e12   
```

**Comment by TristanCacqueray** [comment]
It would be good to know what elastic return for such old date. I guess we should cap the min bound value.

**Comment by morucci** [comment]
This has been fixed via #619
